src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCNV.java

```diff
@@ -135,7 +139,7 @@
             shortName = NUM_BURN_IN_ALLELE_FRACTION_SHORT_NAME,
             optional = true
     )
-    protected int numBurnInAlleleFraction = 50;
+    protected int numBurnInAlleleFraction = 100;
```

Comment:
no real reason, i just decided to switch the defaults to the values i've been using for evaluation.  probably a bit overkill, but runtime is still reasonable (with merging CR neutral, typically ~5-15 minutes for LUAD, max ~30 minutes).